src/fswAlgorithms/transDetermination/ephemDifferenceWithUncertainty/_UnitTest/test_ephemDifferenceWithUncertainty.py

```diff
@@ -35,10 +35,10 @@
                           ([0., 20., -30.], [40., 0., 60.], [-70., -80., 0.], [0., 110., 120.], 5., 5.),
                           ])
 def test_ephem_difference_with_uncertainty(show_plots, r_1_N, r_2_N, v_1_N, v_2_N, sig_1_N, sig_2_N):
-    ephem_difference_with_uncertainty_test_function(show_plots, r_1_N, r_2_N, v_1_N, v_2_N, sig_1_N, sig_2_N)
+    ephem_difference_with_uncertainty_test_function(r_1_N, r_2_N, v_1_N, v_2_N, sig_1_N, sig_2_N)
```

Comment:
:small_red_triangle_down: You can remove this function call indirection. Just have the `test_ephem_difference_with_uncertainty` function.